dockerfiles/grist/ressources/custom.css

```diff
@@ -57,3 +58,35 @@
   background-color: #16b378;
   border-color: #16b378;
 }
+/* END Custom ACL colors */
+
+/* START Custom document card in home page colors */
+.test-dm-public {
+  --icon-color: #cecece !important;
+}
+
+.test-dm-pinned-initials {
+  border-color: #cecece !important;
+}
+/* END Custom document card in home page colors */
+
+/* START Custom tooltip colors */
+.test-tooltip {
```

Comment:
I have just discovered that the undo popup has the same issue:

![image](https://github.com/numerique-gouv/dockerfiles/assets/371705/e3243a77-f311-41db-8257-68f24b7610ab)


You may change the selector to bring the same change to it

```suggestion
.test-tooltip, .test-notifier-toast-wrapper {
```